### DISCUSSION TITLE
[FEATURE-REQUEST] Comportamiento no esperado en visualización condicional

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#286] [3. Ideas & Feature Requests] [answered]
Buenas,

Esto parecerá prueba de lógica pero dejaré un video para demostrar el comportamiento para hacerlo un poco más explicativo.

Tengo una question (L) de tipo lista que es la que ejecuta el llenado de otra question (T) de tipo texto, que cambia según lo que se seleccione en L. T comanda la visualización de otra question (Q) de tipo lista. Esto es: L cambia a T, que comanda a Q.

Al marcar una opción en L **que haga cumplir la condición** para que se visualice Q, si la selección en L se hizo después de haber desmarcado las alternativas en L, no se muestra Q. Luego que falló la visualización de este modo, sí se muestra Q como se espera.

Video del bug
https://user-images.githubusercontent.com/12589784/123015808-35bc7e00-d397-11eb-98a8-e2bc01299ac6.mov

Saludos.

**Comment by NicolasPLG** [comment]
A mi me ha aparecido algo similar: que el iscomanded de la question no me aparece y debo guardar nuevamente la question, para que se me pueda visualizar

**Comment by guillevalin** [comment, marked as answer]
@SeIturra una consulta: El llenado de "T" que comanda a "Q" es realizado mediante questionExec?

Si la respuesta es positiva, es una funcionalidad no oficialmente soportada por la aplicación, la cual podría funcionar (o no) dado que habrán condiciones de carrera (se ejecuta el questionExec primero, funcionara el isCommanded, en caso contrario, no funcionará).

En caso de que sea algo crítico necesario para un proyecto, tendrán que solicitar un feature request a través de su representante Cotalker haciendo referencia a la tarea #3099.
